### PR TITLE
Doc and error message tweaks

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: man/systemd-tmpfiles.xml [hunk 1]
```diff
@@ -65,6 +65,19 @@
...
+    <filename>systemd-tmpfiles-clean.service</filename>) invoke <command>systemd-tmpfiles</command> to create
+    system files and to perform system wide cleanup. Those services read administrator-controlled
+    configuration files in <filename>tmpfiles.d/</filename> directories. User services
+    (<filename>systemd-tmpfiles-setup.service</filename>,
+    <filename>systemd-tmpfiles-clean.service</filename>) also invoke <command>systemd-tmpfiles</command>, but
+    it reads separate set of files, which includes user-controlled files under
+    <filename>~/.config/user-tmpfiles.d/</filename> and <filename>~/.local/share/user-tmpfiles.d/</filename>,
+    and administrator-controller files under <filename>/usr/share/user-tmpfiles.d/</filename>. Users may use
+    this to create and clean up files under their control, but the system instance performs global cleanup
+    and is not influenced by user configuration.</para>
```
Comment (poettering) [inline man/systemd-tmpfiles.xml]: i'd be more explicit here. maybe add: "Note that this means a time-based clean-up configured in the system instance, such as the one typically configured for /tmp, will thus also affect files created by the user instance if they are placed in /tmp, even if the user instance's time-based clean-up is turned off."

File: src/core/main.c [hunk 2]
```diff
@@ -1546,6 +1546,11 @@ static int fixup_environment(void) {
         if (setenv("TERM", t, 1) < 0)
                 return -errno;
 
+        /* The kernels sets HOME=/ for init. Let's undo this. */
+        if (streq_ptr(getenv("HOME"), "/") &&
```
Comment (poettering) [inline src/core/main.c]: I think `streq_ptr()` → `path_equal_ptr()` would make this prettier.

File: src/core/main.c [hunk 3]
```diff
@@ -1546,6 +1546,11 @@ static int fixup_environment(void) {
...
+            setenv("HOME", "/root", 1) < 0)
+                log_warning_errno(errno, "Failed to set $HOME: %m");
```
Comment (poettering) [inline src/core/main.c]: i must say, I'd probably just unset the whole env var if we are PID 1

File: man/systemd.unit.xml [hunk 4]
```diff
@@ -1607,7 +1607,9 @@
           <row>
             <entry><literal>%h</literal></entry>
             <entry>User home directory</entry>
-            <entry>This is the home directory of the user running the service manager instance. In case of the system manager this resolves to <literal>/root</literal>.</entry>
+            <entry>This is the home directory of the <emphasis>user running the service manager instance</emphasis>. In case of the system manager this resolves to <literal>/root</literal>.
+
+Note that this setting is <emphasis>not</emphasis> influenced by the <varname>User=</varname> setting.</entry>
```
Comment (poettering) [inline man/systemd.unit.xml]: weird indentation

Comment (poettering) [inline man/systemd.unit.xml]: … by the User= setting configurable in the [Service] section of the a service unit.

File: man/systemd.unit.xml [hunk 5]
```diff
@@ -1697,12 +1699,16 @@
           <row>
             <entry><literal>%u</literal></entry>
             <entry>User name</entry>
-            <entry>This is the name of the user running the service manager instance. In case of the system manager this resolves to <literal>root</literal>.</entry>
+            <entry>This is the name of the <emphasis>user running the service manager instance</emphasis>. In case of the system manager this resolves to <literal>root</literal>.
+
+Note that this setting is <emphasis>not</emphasis> influenced by the <varname>User=</varname> setting.</entry>
```
Comment (poettering) [inline man/systemd.unit.xml]: dito

File: man/systemd.unit.xml [hunk 6]
```diff
@@ -1697,12 +1699,16 @@
...
           </row>
           <row>
             <entry><literal>%U</literal></entry>
             <entry>User UID</entry>
-            <entry>This is the numeric UID of the user running the service manager instance. In case of the system manager this resolves to <literal>0</literal>.</entry>
+            <entry>This is the numeric UID of the <emphasis>user running the service manager instance</emphasis>. In case of the system manager this resolves to <literal>0</literal>.
+
+Note that this setting is <emphasis>not</emphasis> influenced by the <varname>User=</varname> setting.</entry>
```
Comment (poettering) [inline man/systemd.unit.xml]: dito

File: src/core/main.c (hunk 3 above)
Comment (keszybz) [reply src/core/main.c]: Yeah, I wasn't sure what the best option is. Let's unset it.

File: man/systemd.unit.xml (hunk 4 above)
Comment (keszybz) [reply man/systemd.unit.xml]: That's docbook for you. This is within `<entry>` and it needs to be like that to look correct under both output formats.

**Comment by keszybz** [comment]
Updated with the suggested changes.
(Note: I pushed on more commit shortly before your review. Not sure if you saw that.)

File: man/systemd-tmpfiles.xml [hunk 7]
```diff
@@ -65,6 +65,22 @@
...
+    system files and to perform system wide cleanup. Those services read administrator-controlled
+    configuration files in <filename>tmpfiles.d/</filename> directories. User services
+    (<filename>systemd-tmpfiles-setup.service</filename>,
+    <filename>systemd-tmpfiles-clean.service</filename>) also invoke <command>systemd-tmpfiles</command>, but
+    it reads a separate set of files, which includes user-controlled files under
+    <filename>~/.config/user-tmpfiles.d/</filename> and <filename>~/.local/share/user-tmpfiles.d/</filename>,
+    and administrator-controller files under <filename>/usr/share/user-tmpfiles.d/</filename>. Users may use
+    this to create and clean up files under their control, but the system instance performs global cleanup
+    and is not influenced by user configuration. Note that this means a time-based cleanup configured in the
+    system instance, such as the one typically configured for /tmp, will thus also affect files created by
```
Comment (poettering) [inline man/systemd-tmpfiles.xml]: `/tmp` → `<filename>/tmp</filename>`

File: man/systemd-tmpfiles.xml [hunk 8]
```diff
@@ -65,6 +65,22 @@
...
+    the user instance if they are placed in /tmp, even if the user instance's time-based cleanup is turned
```
Comment (poettering) [inline man/systemd-tmpfiles.xml]: dito

**Comment by keszybz** [comment]
Updated.

**Pull request merged**